code/lib/telemetry/src/storybook-metadata.ts

```diff
@@ -69,6 +69,32 @@ export const computeStorybookMetadata = async ({
     };
   }
 
+  const testPackages = [
+    'playwright',
+    'vitest',
+    'jest',
+    'cypress',
+    'nightwatch',
+    'webdriver',
+    '@web/test-runner',
+    'puppeteer',
+    'karma',
+    'jasmine',
+    'chai',
+    'testing-library',
+    '@ngneat/spectator',
+    'wdio',
+    'msw',
+  ];
+  const testPackageDeps = Object.keys(allDependencies).filter((dep) =>
+    testPackages.find((pkg) => dep.includes(pkg))
+  );
+  metadata.testPackages = Object.fromEntries(
+    await Promise.all(
+      testPackageDeps.map(async (dep) => [dep, (await getActualPackageVersion(dep))?.version])
```

Comment:
You might want to filter the packages out like anything that includes plugin, preset or addon. Else you might get a bunch of noise